### PR TITLE
Updating most of the missing icons

### PR DESCRIPTION
**Pull request by pyronicide**

### CODE REVIEW

**Comment by pyronicide** [comment]
@jsancio

**Comment by jsancio** [comment]
@pyronicide CI is failing:
```
jsonschema: error: argument -i/--instance: invalid _json_file value: '/tmp/universe/scripts/../repo/packages/S/swarm/0/resource.json'
```

File: repo/packages/S/swarm/0/resource.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "images": {
-    "icon-small": "https://downloads.mesosphere.com/swarm/assets/icon-service-swarm-large.png",
-    "icon-medium": "https://downloads.mesosphere.com/swarm/assets/icon-service-swarm-large.png",
-    "icon-large": "https://downloads.mesosphere.com/swarm/assets/icon-service-swarm-large.png"
-  },
+    "icon-small": "https://downloads.mesosphere.com/universe/assets/icon-service-swarm-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/universe/assets/icon-service-swarm-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/universe/assets/icon-service-swarm-large.png"
+  }
```
Comment (jsancio) [inline repo/packages/S/swarm/0/resource.json]: missing a comma `,`.

File: repo/packages/R/riak/0/resource.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "images": {
-    "icon-small": "http://riak-tools.s3.amazonaws.com/riak-mesos/riak-mesos-small.png",
-    "icon-medium": "http://riak-tools.s3.amazonaws.com/riak-mesos/riak-mesos-medium.png",
-    "icon-large": "http://riak-tools.s3.amazonaws.com/riak-mesos/riak-mesos-large.png",
+    "icon-small": "https://downloads.mesosphere.com/universe/assets/icon-service-riak-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/universe/assets/icon-service-riak-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/universe/assets/icon-service-riak-large.png"
```
Comment (jsancio) [inline repo/packages/R/riak/0/resource.json]: missing a comma `,`

**Comment by jsancio** [comment]
Two minor comments. Looks good otherwise.

**Comment by jsancio** [comment]
Also, we now have marathon-lb icons. They are attached to the Jira.

**Comment by pyronicide** [comment]
@jsancio I think I got everything.

**Comment by jsancio** [comment]
LGTM! Thanks!

**Pull request merged**